src/lib/CRSF/CRSF.cpp

```diff
@@ -79,7 +79,7 @@ uint32_t CRSF::RequestedRCpacketInterval = 5000; // default to 200hz as per 'nor
 volatile uint32_t CRSF::RCdataLastRecv = 0;
 volatile int32_t CRSF::OpenTXsyncOffset = 0;
 bool CRSF::OpentxSyncActive = true;
-uint32_t CRSF::OpenTXsyncOffsetSafeMargin = 4000; // 400us
+uint32_t CRSF::OpenTXsyncOffsetSafeMargin = 1000; // 200us
```

Comment:
That would be 100us

Reply:
Yep I updated it to 100us as all of the testing from myself and Wez looked good.

Reply:
The comment need to say 100us

Reply:
Argh crap.